src/styles/Home.css

```diff
@@ -0,0 +1,48 @@
+
+/* .banner-image{
+    background-image: url(/assets/gourmet\ bg.jpg);
+    height: calc(100vh - 5rem);
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    position: relative;
+
+} */
```

Comment:
remove the commented code

Reply:
thanks for pointing it out will change it.